src/main/java/edu/njit/jerse/automation/RepositoryAutomationEngine.java

```diff
@@ -0,0 +1,232 @@
+package edu.njit.jerse.automation;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.eclipse.jgit.api.errors.GitAPIException;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+// TODO: Add this functionality to the README
```

Comment:
I'd prefer that we do this now, since if we don't it's likely that we'll forget about it for an arbitrarily-long amount of time. Even some simple documentation, like the CSV format and the command to run to invoke this (with a simple example) would be enough.

Reply:
Also, the format of the URLs in the "Repository" column of the CSV should be documented.

Reply:
Please see the README updates for this.